inst/pkgdown/BS5/_pkgdown.yml

```diff
@@ -24,10 +24,10 @@ authors:
     html: <img src='https://www.tidyverse.org/rstudio-logo.svg' alt='RStudio' width='72' />
   Posit, PBC:
     href: https://www.posit.io
-    html: <img src='https://www.tidyverse.org/posit-logo.svg' alt='Posit' height='16' style="margin-bottom: 3px;" />
+    html: <img src='https://www.tidyverse.org/posit-logo.svg' alt='Posit' height='16' style="margin-bottom:3px;" />
```

Comment:
An alternate fix using multiline yaml (my preference because it's less susceptible to these kinds of parsing details).

```suggestion
    html: >-
      <img src='https://www.tidyverse.org/posit-logo.svg' alt='Posit' height='16' style="margin-bottom: 3px;" />
```

Reply:
Or quote the whole line?

Reply:
Yeah that works too (but then you have to worry about `'` vs `"` or escaping quotes). Anyway, you have all the options now! 😄 

```suggestion
    html: '<img src="https://www.tidyverse.org/posit-logo.svg" alt="Posit" height="16" style="margin-bottom: 3px;" />'
```